python/Makefile

```diff
@@ -17,12 +17,14 @@
 DEBUG           ?= 0		# Include debugging symbols
 MAX_DIM         ?= 4		# Maximum number of dimensions
 OUTPUT_LEVEL    ?= LEVEL_DEBUG	# Compile time logging level
-USE_CUDA        ?= 1		# Include CUDA support (requires CUDA)
-USE_GASNET      ?= 0		# Include GASNet support (requires GASNet)
+USE_CUDA        ?= 0		# Include CUDA support (requires CUDA)
+USE_GASNET      ?= 1		# Include GASNet support (requires GASNet)
 USE_HDF         ?= 0		# Include HDF5 support (requires HDF5)
 ALT_MAPPERS     ?= 0		# Include alternative mappers (not recommended)
-#FF_USE_NCCL  ?= 1
-#USE_CUDART_HIJACK ?= 0
+USE_HIP         ?= 1
+HIP_TARGET      ?= CUDA
+USE_GPU_REDUCTIONS ?= 0
+FF_USE_NCCL  ?= 0
```

Comment:
Same as the above

Reply:
Ops, thanks for pointing it out. I will fix it.